org_fedora_oscap/data_fetch.py

```diff
@@ -10,7 +10,12 @@
 import pycurl
 
 from pyanaconda.core.configuration.anaconda import conf
+from pyanaconda.core import constants
+from pyanaconda.threading import threadMgr, AnacondaThread
+from pyanaconda.modules.common.constants.services import NETWORK
 
+from org_fedora_oscap import common
+from org_fedora_oscap.common import _
```

Comment:
I'm getting an error that it cant import name '_'.

Reply:
I have tested the code, and it doesn't manifest this issue - and `_` is even defined in `common`. How do you bump into this error?

Reply:
I run a kickstart installation with a broken URL in the kickstart. But I can't reproduce this problem anymore. It's possible that I did a mistake on Friday. Please disregard.